featurebyte/api/feature.py

```diff
@@ -128,6 +128,10 @@ def _get_feature_tiles_specs(self) -> List[Tuple[str, List[TileSpec]]]:
         tile_specs = ExtendedFeatureModel(**self.dict(by_alias=True)).tile_specs
         return [(str(self.name), tile_specs)] if tile_specs else []
 
+    @property
+    def row_index_lineage(self) -> Tuple[str, ...]:
+        return tuple()
```

Comment:
Would be good to leave some comment why it is empty. Is it because we did not use this value directly? Curious why don't we generate it from the operation structure.

Reply:
Sure, will add some comments. The intention is to make row index lineage check always pass for Feature objects, so the return value here can be any value as long as it's a constant.

I think there are a few possible approaches:

1. Update all places where we check whether `row_index_lineage` matches to ignore Feature 
2. Update derivation of `row_index_lineage` in operation structure to return a constant for Feature
3. Override the `row_index_lineage` for Feature class to return a constant (current approach)

Since 1 and 2 likely require more changes in multiple places, I opted the current approach.